app/Http/Controllers/Entity/PostMoveController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Http\Controllers\Entity;
+
+use App\Exceptions\TranslatableException;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\MovePostRequest;
+use App\Http\Requests\TransformEntityRequest;
+use App\Models\Entity;
+use App\Facades\CampaignLocalization;
+use App\Models\EntityNote;
+use App\Services\EntityNoteService;
+use App\Traits\GuestAuthTrait;
+use Illuminate\Support\Facades\Auth;
+
+class PostMoveController extends Controller
+{
+    /**
+     * Guest Auth Trait
+     */
+    use GuestAuthTrait;
+
+    /** @var EntityNoteService */
+    protected $service;
+
+    /**
+     * AbilityController constructor.
+     * @param EntityNoteService $service
+     */
+    public function __construct(EntityNoteService $service)
+    {
+        $this->service = $service;
+    }
+
+    /**
+     * @param Entity $entity
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     * @throws \Illuminate\Auth\Access\AuthorizationException
+     */
+    public function index(Entity $entity, EntityNote $entityNote)
+    {
+        $this->authorize('view', $entity->child);
+        $campaign = CampaignLocalization::getCampaign()->entities()->get();
+
+        return view('entities.pages.move.entity_notes.index', compact(
+            'entity',
+            'entityNote',
+            'campaign',
+        ));
+    }
+
+    /**
+     * @param TransformEntityRequest $request
+     * @param Entity $entity
+     * @return \Illuminate\Http\RedirectResponse
+     * @throws \Illuminate\Auth\Access\AuthorizationException
+     */
+    public function move(MovePostRequest $request, Entity $entity, EntityNote $entityNote)
+    {
+
```

Comment:
Missing permission checks, an unlogged user or one without the correct permission could call this page right now :-)